stubs/docker/docker/api/container.pyi

```diff
@@ -157,7 +157,7 @@ class ContainerApiMixin:
     def start(self, container: _Container, *args, **kwargs) -> None: ...
```

Comment:
At https://github.com/docker/docker-py/blob/a3652028b1ead708bd9191efb286f909ba6c2a49/docker/api/container.py#L1128-L1133 we see:

```python
        if args or kwargs:
            raise errors.DeprecatedMethod(
                'Providing configuration in the start() method is no longer '
                'supported. Use the host_config param in create_container '
                'instead.'
            )
```

Tips are welcome on how to change this signature and not break CI.

Reply:
You can just remove the arguments and then add the function to `@tests/stubtest_allowlist.txt` (with a comment) to silence stubtest.

Reply:
Thank you @srittau , I have done this.